src/Views/LockPanel.vala

```diff
@@ -28,6 +33,19 @@ public class SecurityPrivacy.LockPanel : Granite.SimpleSettingsPage {
     }
 
     construct {
+        permission.notify["allowed"].connect (() => {
+            try {
+                fprint = Bus.get_proxy_sync (BusType.SYSTEM, DBUS_FPRINT_NAME, DBUS_FPRINT_PATH, DBusProxyFlags.NONE);
```

Comment:
@vjr Im getting error here, connecting to DBUS

Reply:
What  is the error details you get?

I assume you have the libfprint and libfprint-dev packages installed and your fingerprint device is actually supported?

I'm sorry I wanted to try your branch but I sold my old laptop which was great to test this and my new laptop fingerprint reader is not yet supported.

Reply:
Yes libfprint and libfprint-dev are installed and my fingerprint works fine. 
I am able to set up from comand line, actually I can see the DBUS info into D-Feet app but cant connect to them with 
``` Bus.get_proxy_sync () ```

Reply:
@vjr 
![Screenshot from 2021-08-16 11 22 44](https://user-images.githubusercontent.com/22928302/129596556-1b873b60-8531-4cbf-b7cb-b3202d58068e.png)

Reply:
Are you getting any error in catch block if you log it or it just hangs?

Try path ending with /Fprint/Manager maybe?

Reply:
Hello again... I just looked at gnome control center git code and it may be path should be ending with  /Fprint/Manager  most likely... hopefully it works for you too?

Reply:
Ahmm, let me check.

Reply:
```
public const string DBUS_FPRINT_NAME = "net.reactivated.Fprint.Manager";
public const string DBUS_FPRINT_PATH = "/net/reactivated/Fprint/Manager";
``` 
Getting this error
```
Connection to Fprint DBus  establishederrorGDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name net.reactivated.Fprint.Manager was not provided by any .service files
```

Reply:
Hi ... I meant only the PATH should be changed not the NAME... try that?

Reply:
So...

`public const string DBUS_FPRINT_NAME = "net.reactivated.Fprint";`

and

`public const string DBUS_FPRINT_PATH = "/net/reactivated/Fprint/Manager";`

Reply:
Yes!, it works. 
Thanks a Lot